api/src/test/resources/org/openmrs/module/ugandaemr/include/dsdmTestDataSet.xml

```diff
@@ -46,18 +48,12 @@
     <concept_answer  concept_answer_id="7089"  concept_id="165143"  answer_concept="165141"  creator="1"  date_created="2018-10-15 13:02:40"  sort_weight="4"  uuid="0a2ac1f3-69ca-4b7d-99ba-0f55a512fa29"/>
     <concept_answer  concept_answer_id="7090"  concept_id="165143"  answer_concept="165142"  creator="1"  date_created="2018-10-15 13:02:40"  sort_weight="5"  uuid="cf6cea6f-c3e7-49f6-a447-15613f36504e"/>
 
-    <visit_type  visit_type_id="1"  name="Facility Visit"  description="Patient visits the clinic/hospital (as opposed to a home visit, or telephone contact)"  creator="1"  date_created="2013-08-02 03:39:43"  changed_by="1"  date_changed="2016-08-30 03:52:10"  retired="0"  uuid="7b0f5697-27e3-40c4-8bae-f4049abfb4ed"/>
```

Comment:
@ssmusoke why did you delete visit type facility  from the data set.

Reply:
@slubwama Replaced, I had removed it when including the StandardDataset but there was too much duplication